server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/BasicServlet.java

```diff
@@ -264,4 +264,12 @@ protected static void banner(StringBuilder sb, String klass, String text) {
     sb.append("<br />\n<h2 class='").append(klass).append("'>").append(text).append("</h2>\n");
   }
 
+  /**
+   * Creates a {@link Cookie} with the given name and value, also setting the HttpOnly attribute on the cookie.
+   */
+  protected static Cookie createCookie(String name, String value) {
+    Cookie c = new Cookie(name, value);
+    c.setHttpOnly(true);
```

Comment:
We may only be using these server-side. Pretty sure we're refreshing the page and getting the resorted version from the server. I just wasn't sure if that was always the case, or if there was some fringe place where it would break.